integration-test/helpers/mocks.webkit.js

```diff
@@ -244,6 +244,7 @@ export function createWebkitMocks(platform = 'macos') {
                                     password_generation: true,
                                     credentials_saving: true,
                                     inlineIcon_credentials: true,
+                                    partial_form_saves: true,
```

Comment:
Setting this to true means https://github.com/duckduckgo/duckduckgo-autofill/blob/main/integration-test/tests/save-prompts.ios.spec.js and https://github.com/duckduckgo/duckduckgo-autofill/blob/main/integration-test/tests/save-prompts.macos.spec.js are working as expected!